### ISSUE TITLE
Missing statistics tracking in normalization layers

### ISSUE THREAD

**Issue by avik-pal** [LuxDL/Lux.jl#780]
- `InstanceNorm` (https://pytorch.org/docs/stable/generated/torch.nn.InstanceNorm2d.html): default is `false`, so it can be done

We need to extend the interface of `LuxLib.instancenorm` to take in statistics to feed into `_normalization`.
